tema-html-1/Index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Calendar</title>
+  </head>
+  <body>
+    <h2>My schedule</h2>
+    <em>May 2019</em>
+    <progress value="80" max="100">70%</progress>
+    <br />
+
+    <a href="http://127.0.0.1:5500/tema-html-1/Bio.html"> Bio</a>
+
+    <a href="http://127.0.0.1:5500/tema-html-1/Blog.html">Blog</a>
```

Comment:
in general, pentru a naviga prin alte pagini din interiorul aceluiasi proiect este indeajuns sa pui path-ul relativ catre fisier(ex: `/tema-html-1/Bio.html`), fara a fi nevoie sa adaugi si [domeniul](https://www.techtarget.com/whatis/definition/domain) (`http://127.0.0.1:5500`)

```suggestion
    <a href="/tema-html-1/Bio.html"> Bio</a>

    <a href="/tema-html-1/Blog.html">Blog</a>
```

Reply:
Eu daca vreau sa vad fisierele fara a pornii live server, paginile nu imi vor aparea astfel 😄